packages/api/src/command/webhook/webhook.ts

```diff
@@ -92,6 +87,13 @@ export const processRequest = async (
     });
   };
 
+  const { webhookUrl, webhookKey, webhookEnabled } = settings;
+  if (!webhookUrl || !webhookKey) {
+    sendAnalytics("failure");
```

Comment:
The counts were off in Posthog because we were missing whs that were in the db but could get sent because the cx had no wh settings